website/docs/r/pi_shared_processor_pool.html.markdown

```diff
@@ -41,45 +44,49 @@ resource "ibm_pi_shared_processor_pool" "testacc_shared_processor_pool" {
-- `pi_shared_processor_pool_placement_group_id` - (Optional, String) The ID of the placement group the shared processor pool is created in.
+Review the argument references that you can specify for your resource.
+
+* `pi_cloud_instance_id` - (Required, String) The GUID of the service instance associated with an account.
+* `pi_host_id` - (Optional, String) The host id of a host in a host group (only available for dedicated hosts).
+* `pi_shared_processor_pool_host_group` - (Required, String) Host group of the shared processor pool. Valid values are 's922', 'e980' and 's1022'.
+* `pi_shared_processor_pool_name` - (Required, String) The name of the shared processor pool.
+* `pi_shared_processor_pool_reserved_cores` - (Required, Integer) The amount of reserved cores for the shared processor pool.
+* `pi_shared_processor_pool_placement_group_id` - (Optional, String) The ID of the placement group the shared processor pool is created in.
 
 ## Attribute reference
+
  In addition to all argument reference list, you can access the following attribute reference after your resource is created.
 
-- `allocated_cores` - (Float) The allocated cores in the shared processor pool.
-- `available_cores` - (Integer) The available cores in the shared processor pool.
-- `host_id` - (Integer) The host ID where the shared processor pool resides.
-- `instances` - (List of Map) The list of server instances that are deployed in the shared processor pool.
+* `allocated_cores` - (Float) The allocated cores in the shared processor pool.
+* `available_cores` - (Integer) The available cores in the shared processor pool.
+* `host_id` - (Integer) The host ID where the shared processor pool resides.
+* `instances` - (List of Map) The list of server instances that are deployed in the shared processor pool.
+  
   Nested scheme for `instances`:
-  - `availability_zone` - (String) Availability zone for the server instances.
-  - `cpus` - (Integer) The amount of cpus for the server instance.
-  - `id` - (String) The server instance ID.
-  - `memory` - (Integer) The amount of memory for the server instance.
-  - `name` - (String) The server instance name.
-  - `status` - (String) Status of the instance.
-  - `uncapped` - (Bool) Identifies if uncapped or not.
-  - `vcpus` - (Float) The amout of vcpus for the server instance.
-- `shared_processor_pool_id` - (String) The shared processor pool's unique ID.
-- `status` - (String) The status of the shared processor pool.
-- `status_detail` - (String) The status details of the shared processor pool.
+  * `availability_zone` - (String) Availability zone for the server instances.
+  * `cpus` - (Integer) The amount of cpus for the server instance.
+  * `id` - (String) The server instance ID.
+  * `memory` - (Integer) The amount of memory for the server instance.
+  * `name` - (String) The server instance name.
+  * `status` - (String) Status of the instance.
+  * `uncapped` - (Bool) Identifies if uncapped or not.
```

Comment:
```suggestion
  * `uncapped` - (Boolean) Identifies if uncapped or not.
```

Are we changing all references to Bool to Boolean?

Reply:
good catch!